guide/index.html

```diff
@@ -0,0 +1,291 @@
+    <link type="text/css" rel="stylesheet" href="../res/reset.css"/>
+    <link type="text/css" rel="stylesheet" href="../res/style.css"/>
+    <link type="text/css" rel="stylesheet" href="../res/monokai.css"/>
+</head>
+<body>
+<div class="screen title-screen">
+    <div class="main-title">
+        <h1>
+            Guide
+        </h1>
+        <small>
+            Step-by-step introduction to Pencil.js mastery&nbsp;&hellip;
+        </small>
+    </div>
+    <div class="element description">
+        <p>
+            In this guide, you will discover <strong>most of the features</strong> of the Pencil.js library.<br/>
+            It assumes <strong>basic knowledge</strong> of web technologies (HTML, JavaScript, NPM &hellip;).<br/>
+            by the end of the page, you will be able to build your own <strong>creations with ease</strong>.
+        </p>
+    </div>
+</div>
+<div class="screen">
+    <h2>Getting started</h2>
+    <p>
+        First of all, you need to add the library to your page.<br/>
+        You can quickly doing so by using a CDN:
+    </p>
+    <pre><code class="xml">
+&lt;script src="https://unpkg.com/pencil.js"&gt;&lt;/script&gt;
+&lt;!-- or --&gt;
+&lt;script src="https://cdn.jsdelivr.net/npm/pencil.js"&gt;&lt;/script&gt;
+    </code></pre>
+    <p>
+        Or you can load it with NPM:
+    </p>
+    <pre><code>
+$ npm install pencil.js
+        </code></pre>
+    <pre><code class="javascript">
+import Pencil from "pencil.js";
+    </code></pre>
+    <p>
+        In either case, you can now use it:
+    </p>
+    <pre><code class="javascript">
+console.log(Pencil.version);
+    </code></pre>
+</div>
+<div class="screen">
+    <h2>Scene</h2>
+    <p>
+        The first and main element of your code should be a {Scene}.
+        A scene is bound to an element or to the whole screen by default.
+    </p>
+    <pre><code class="javascript">
+const myFirstScene = new Pencil.Scene();
+// or
+const myFirstScene = new Pencil.Scene(wrapperElement);
+    </code></pre>
+    <p>
+        When your {Scene} is ready, if you only need a still frame you can render it once.<br/>
+        Or, you can make it alive by refreshing every frames.
+    </p>
+    <pre><code class="javascript">
+// Draw the scene once
+myFirstScene.render();
+// or refresh every frame
+myFirstScene.startLoop();
+    </code></pre>
+</div>
+<div class="screen">
+    <h2>Components</h2>
+    <p>
+        {Component} refers to any drawable part of a scene.
+        As an abstract class, it should not be instantiated by itself, but by any sub-class (ex: Rectangle, Star, Image &hellip;).
+    </p>
+    <pre><code class="javascript">
+const myRectangle = new Pencil.Rectangle();
+    </code></pre>
+    <p>
+        All sub-class constructor parameters are as follows: <code class="inline">(position, [...a number of specific properties], options)</code>.<br/>
+        To continue on our previous example:
+    </p>
+    <pre><code class="javascript">
```

Comment:
That would be nice to see the result. Even if that's already in the presentation page.

Reply:
I don't know how to do it. It's a good idea, but too much preview would impair reading flow.